flyway/scxa/migrations/V14__scxa_add_json_coords_param.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scxa_coords
+ADD COLUMN json_parameterisation JSONB DEFAULT NULL;
```

Comment:
I'm thinking that we don't want this to be NULL ever, right? This should always have a value, so it should be a not null. Or maybe we can make it non-null once we have added the data via a query and we remove the other parameter field.

Reply:
In a subsequent PR I mean.

Reply:
Our JSON seralising library of choice, GSON, handles null values gracefully, so I think we don’t need to worry about it.

Reply:
The latter was my thinking, yes.

Reply:
I’ve also had a quick look at Jackson, the most popular JSON-parsing library and as expected it handles null values well, both serialising and deserialising (remember that `null` is now valid JSON).

Reply:
It probably shouldn't be null though, if it ended up that way it would probably indicate a loading error. I'll fix it as NOT NULL once we populate it.